azure-pipelines.yml

```diff
@@ -114,7 +114,7 @@ stages:
           displayName: NPM tests
           workingDirectory: "$(Build.SourcesDirectory)/src/Microsoft.DotNet.Interactive.Js"
 
-        - script: npm i
+        - script: npm install
           displayName: Install NPM packages for VS Code extension
```

Comment:
Since we have the need for powershell is there a way to ensure powershell presence and also for node to latest lts for our builds too?